### DISCUSSION TITLE
How do I add an example?

### DISCUSSION THREAD

**Discussion by Jared-Dahlke** [vercel/next.js#46641] [Help]
### Summary

I saw the guidelines in contributing, but how specifically do I actually add an example? Make a PR?

**Comment by icyJoseph** [comment]
- https://github.com/vercel/next.js/blob/canary/contributing/examples/adding-examples.md

And pretty much yes, open a PR, it'll be tagged by a maintainer as `examples`, and reviewed by someone who's in charge of that.
